File: api/models/football-data/team.go
package footballdata

type Team struct {
	Area                Area          `json:"area"`
	ID                  int           `json:"id"`
	Name                string        `json:"name"`
	ShortName           string        `json:"shortName"`
	Tla                 string        `json:"tla"`
	Crest               string        `json:"crest"`
	Address             string        `json:"address"`
	Website             string        `json:"website"`
	Founded             int           `json:"founded"`
	ClubColors          string        `json:"clubColors"`
	Venue               string        `json:"venue"`
	RunningCompetitions []Competition `json:"runningCompetitions"`
	Coach               Person        `json:"coach"`
	MarketValue         int           `json:"marketValue"`
	Squad               []Person      `json:"squad"`
	Staff               []Person      `json:"staff"`
	LastUpdated         string        `json:"lastUpdated"`
}

File: ui/predictball/src/app/models/team.ts
import { Area } from "./area";
import { Competition } from "./competition";
import { Person } from "./person";

export interface Team {
	area: Area;
	id: number;
	name: string;
	shortName: string;
	tla: string;
	crest: string;
	address: string;
	website: string;
	founded: number;
	clubColors: string;
	venue: string;
	runningCompetitions: Competition[];
	coach: Person;
	marketValue: number;
	squad: Person[];
	staff: Person[];
	lastUpdated: string;
}